src/options.ts

```diff
@@ -18,6 +18,12 @@ export interface Options {
    */
   jsii?: JsiiOutputOptions;
 
+  /**
+   * Package name
+   * @default - hash of the basepath to the module
+   */
+  packageName?: string
```

Comment:
I am not sure that's the correct name for this because users don't really interact with this value after code generation. Maybe `moduleKey` or something like that.

Docstring help needs to indicate that it should be project-unique.